ods_ci/tests/Resources/Page/Pipelines/OpenShiftPipelines.robot

```diff
@@ -0,0 +1,34 @@
+*** Settings ***
+Documentation       Test suite for OpenShift Pipeline
+Resource            ../../../Resources/RHOSi.resource
+Resource            ../../../Resources/ODS.robot
+Resource            ../../../Resources/Common.robot
+Resource            ../../../Resources/Page/ODH/ODHDashboard/ODHDashboard.robot
+Library             DateTime
+Library             ../../../../libs/DataSciencePipelinesAPI.py
+
+
+*** Variables ***
+${REDHAT_OPENSHIFT_PIPELINES_YAML}=            ods_ci/tests/Resources/Files/redhat-openshift-pipelines.yaml
+
+
+*** Keywords ***
+Data Science Pipelines Suite Setup
+    [Documentation]    Data Science Pipelines Suite Setup
+    Set Library Search Order    SeleniumLibrary
+    RHOSi Setup
+    Install Red Hat OpenShfit Pipelines
+
+Install Red Hat OpenShfit Pipelines
+    [Documentation]    Install Red Hat OpenShfit Pipelines
+    Oc Apply    kind=Subscription    src=${REDHAT_OPENSHIFT_PIPELINES_YAML}     namespace=openshift-operators
```

Comment:
You don't need namespace parameter since in your yaml file you have already declared the namespace under metadata section

https://github.com/red-hat-data-services/ods-ci/blob/master/ods_ci/tests/Resources/Files/redhat-openshift-pipelines.yaml